grid/social-media/posts-lookup/maps/twitter.suma

```diff
@@ -0,0 +1,166 @@
+// Twitter API Reference: https://developer.twitter.com/en/docs/twitter-api
+
+profile = "social-media/posts-lookup@1.0"
+provider = "twitter"
+
+map FindByHashtag {
+
+  http GET "/2/tweets/search/recent" {
+    
+    request {
+      query {
+        query = '#' + input.hashtag
+        'tweet.fields' = 'id,text,created_at,attachments'
+        expansions = 'attachments.media_keys'
+        'media.fields' = 'duration_ms,height,media_key,preview_image_url,type,url,width,alt_text'
+        start_time = input.afterDate
+        pagination_token = input.page
+        max_results = 100
```

Comment:
I'm not sure whether it matters, but snapshots are quite long for this profile, could we use input parameter limit to have less records to compare during testing?

Reply:
👍 I will add the page limit parameter in separate PR.